examples/apply-setters/imperative/README.md

```diff
@@ -0,0 +1,26 @@
+# apply-setters: Imperative Example
+
+### Overview
+
+<!-- TODO(phanimarupaka): Populate this and below -->
```

Comment:
why are we adding new examples in this PR? Especially given they're incomplete.

Reply:
I can split the imperative examples out to a separate PR.